README.md

```diff
@@ -39,13 +39,13 @@ return static function (RectorConfig $rectorConfig): void {
 
 Some rules like `StringFormTypeToClassRector` need access to your Symfony container dumped XML. It contains list of form types with their string names, so it can convert them to class references.
 
-How to add it? Check your `/var/cache` directory and find the XML file for your test env. Then add it in `rector.php`:
+How to add it? Check your `var/cache/` directory and find the XML file for your test env. Then add it in `rector.php`:
 
 ```php
 use Rector\Config\RectorConfig;
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->symfonyContainerXml(__DIR__ . '/var/cache/<env>/appProjectContainer.xml');
+    $rectorConfig->symfonyContainerXml(__DIR__ . '/var/cache/test/App_KernelTestDebugContainer.php');
```

Comment:
Here the method is called "xml", not sure it can contain "php" file

Reply:
Oops, fixed.